solcx/utils/types.py

```diff
@@ -1,4 +1,3 @@
-import numbers
 from collections import abc
```

Comment:
we need to look at what these remaining methods are being used for..  having an entire sub-directory to hold these few functions seems overkill.  maybe we can refactor them out?  but that's a topic for another PR anyway.